doc/en/user/source/services/csw/tutorial.rst

```diff
@@ -152,3 +152,23 @@ This should yield the following result::
       </csw:DomainValues>
     </csw:GetDomainResponse>
 
+To request more than the first 10 records or for more complex queries you can use a HTTP POST request with an XML query as the request body. For example, using the maxRecords option in the following request it is possible to return the first 50 layers with "ImageMosaic" in a keyword::
+  
+  http://localhost:8080/geoserver/csw
+  
+Postbody::
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <GetRecords service="CSW" version="2.0.2" maxRecords="50" startPosition="1" resultType="results" outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2" xmlns="http://www.opengis.net/cat/csw/2.0.2" xmlns:csw="http://www.opengis.net/cat/csw/2.0.2" xmlns:ogc="http://www.opengis.net/ogc" xmlns:ows="http://www.opengis.net/ows" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/" xmlns:gml="http://www.opengis.net/gml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/cat/csw/2.0.2 ../../../csw/2.0.2/CSW-discovery.xsd">
+      <Query typeNames="csw:Record">
+      	<ElementSetName typeNames="csw:Record">full</ElementSetName>
+      	<Constraint version="1.1.0">
+      	  <ogc:Filter>
+      	    <ogc:PropertyIsLike wildCard="%" singleChar="_" escapeChar="\">
+      	      <ogc:PropertyName>dc::subject</ogc:PropertyName>

```

Comment:
I don't quite remember the CSW protocol but... why the double column here?

Reply:
I think I put in a double because I thought I needed to escape the : character but I've just tested the post body and it does need to be a single : rather than a double. I've corrected it but I'm just checking that I've not broken the formatting.

Reply:
Okay, I think the change is fine but I might have committed that incorrectly as it has failed the checks now. Is that something I can fix or does that need to be done but someone else with write access?

Reply:
Since it's a small change, if you used the quick fix procedure, it's easier to close this one and start over with a new one.

Reply:
New pull request created here https://github.com/geoserver/geoserver/pull/3821